extensions/applicationinsights-clickanalytics-js/README.md

```diff
@@ -15,23 +24,21 @@ npm install --save @microsoft/applicationinsights-clickanalytics-js @microsoft/a
 ```
 
 ```js
-import { ApplicationInsights } from '@microsoft/applicationinsights-web';
-import { DebugPlugin } from ' @microsoft/applicationinsights-clickanalytics-js';
+import { ApplicationInsights } from "@microsoft/applicationinsights-web";
+import { DebugPlugin } from " @microsoft/applicationinsights-clickanalytics-js";
 
 const clickPluginInstance = new ClickAnalyticsPlugin();
 // Click Analytics configuration
 const clickPluginConfig = {
-    autoCapture : true
-}
+  autoCapture: true,
+};
 // Application Insights Configuration
 const configObj = {
-    instrumentationKey: "YOUR INSTRUMENTATION KEY",
-    extensions: [
-      clickPluginInstance
-    ],
-    extensionConfig: {
-        [clickPluginInstance.identifier] : clickPluginConfig
-    },
+  instrumentationKey: "YOUR INSTRUMENTATION KEY",
+  extensions: [clickPluginInstance],
+  extensionConfig: {
+    [clickPluginInstance.identifier]: clickPluginConfig,
```

Comment:
nit: as above remove trailing comma (older browsers will barf with extra (and missing) comma's)